eos-add-flatpak-gnome-repos

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+# Copyright (C) 2016 Endless Mobile, Inc.
+# Licensed under the GPLv2
+
+# Add the stable GNOME flatpak repositories after an OS upgrade
+
+TMPDIR=`mktemp -d`
```

Comment:
I think you should remove this in a `trap` so that it gets cleaned up even if there's an error below:
```
trap "rm -rf "$TMPDIR" EXIT
```